jekyll/_cci2/executor-types.md

```diff
@@ -23,19 +23,47 @@ Like any set of choices, there are tradeoffs to using one over the other. Here
 
 ## Docker Executor
 
-### Overview
-A build is running in a Docker container. Image for a container is specified in a config file:
+When you choose the `docker` executor, your build will run in a Docker container. You can specify the container image in `.circleci/config.yml`:
 
-``` YAML
+``` yaml
 jobs:
   build:
     docker:
       - image: alpine:3.4
 ```
 
-It's possible to specify many images. In this case all of them will be running in a common network and all the ports they expose will be available on `localhost` from a [main container]( {{ site.baseurl }}/2.0/glossary#main-container).
+### You Should Use The Docker Executor If...
 
-``` YAML
+- you have a self-sufficient application
```

Comment:
I'm not sure that "use cases" section has to be in between of "how to configure" and "how to specify images". Also, I think separating multi-image config from single-image config breaks flow and makes it hard to read